framework/src/modules/interoperability/base_interoperability_store.ts

```diff
@@ -89,7 +89,10 @@ export abstract class BaseInteroperabilityStore {
 	}
 
 	public async setOwnChainAccount(ownChainAccount: OwnChainAccount): Promise<void> {
-		const ownChainAccountStore = this.getStore(this.moduleID, STORE_PREFIX_OWN_CHAIN_DATA);
+		const ownChainAccountStore = this.getStore(
```

Comment:
The verify hook of MessageRecoveryCommand needs access to interoperabilityStore but the return type of `getStore` in CommandVerifyContext is `ImmutableSubStore`. Hence the return type of `getStore`(StoreCallback) here is updatedto be `SubStore | ImmutableStore`. So, for functions other than getters the return type needs to be cast back to `SubStore` since `ImmutableStore` doesn't support them.

Reply:
Then don't we need to make changes in `CommandVerifyContext` instead of casting?

Reply:
No. `CommandVerifyContext` can not be change because on verify step it can only access getter.